src/test/integration/profile_copy.test.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Again an excellent example of an integration test, however it is not testing production code. It's fine for the assignment.